Modelica/Electrical/Spice3.mo

```diff
@@ -4435,6 +4435,12 @@ on the model behaviour.
 </html>"));
   end Types;
 
+  package Constants "Spice3-specific constants"
+    extends Modelica.Icons.Package;
+
+    constant Modelica.Units.SI.Time infiniteTime= 1e20 "Very large time horizon (3e12 years) avoiding overflow in time computations";
```

Comment:
While we are defining new (local) constants, what do you think about _not_ enlarging the scope of the problem of the unfortunate (incorrect) naming of `Modelica.Constants.inf`, and give _this_ constant a more accurate name, e.g. `hugeTime` or `largeTime` or something like that, instead of infinity, which it is not?

Reply:
It's infinite for all practical purposes. Large or huge have no definite meaning, large with respect to what? Infinite time means you'll never get there, so you'll always get a ramp. If you look at the parameter window and you see that both period and width are "infinite" it is clear that there will be no periodic duration.

To me, the problem with inf. is not in the name, it's in the _value_.

But I guess we are over-discussing this issue here...

Reply:
Okay, then.